source/code/scxsystemlib/common/scxostypeinfo.cpp

```diff
@@ -117,7 +117,7 @@ namespace SCXSystemLib
 
     const wstring SCXOSTypeInfoDependencies::getConfigPath() const
     {
-        return L"/etc/opt/microsoft/scx/conf/scxconfig.conf";
+        return SCXSystemLib::OSTypeInfo_GetConfigPath();
```

Comment:
With the change above this will become SCXSystemLib:SCXProductDependencies::OSTypeInfo_GetConfigPath